app/core/speaker_diarization.py

```
@@ -0,0 +1,337 @@
+                if duration >= min_duration:
+                    filtered_intervals.append({
+                        "start": start / sr,
+                        "end": end / sr,
+                        "duration": duration,
+                        "speaker": "SPEAKER_00",
+                        "speaker_id": 0  # fallback도 숫자로 변환
+                    })
+            
+            return filtered_intervals
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)